src/minuteman_route_mgr.erl

```diff
@@ -162,71 +168,77 @@ code_change(_OldVsn, State, _Extra) ->
 %%%===================================================================
 %%% Internal functions
 %%%===================================================================
-
-get_routes(Pid) ->
+get_minuteman_iface(Pid) ->
```

Comment:
There is a gen_netlink function to get interface index given the name. That will simplify you code a lot